spec/fake_braintree/transaction_spec.rb

```diff
@@ -77,6 +87,13 @@
       result = Braintree::Transaction.refund(create_id('foobar'), '1')
       expect(result).to be_success
     end
+
+    it "sets the creation time" do
+      transaction = Braintree::Transaction.refund(create_id('foobar'), '1').transaction
+
+      creation_time = Time.parse(transaction.created_at)
```

Comment:
Do not use `Time.parse` without zone. Use one of `Time.zone.parse`, `Time.parse.current`, `Time.parse.in_time_zone`, `Time.parse.utc`, `Time.parse.getlocal`, `Time.parse.iso8601`, `Time.parse.jisx0301`, `Time.parse.rfc3339`, `Time.parse.to_i`, `Time.parse.to_f` instead.